### PR TITLE
Support importing haploid genotypes as diploid

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: python/hail/context.py [hunk 1]
```diff
@@ -355,6 +355,20 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
                    sites_only=False, store_gq=False, pp_as_pl=False, skip_bad_ad=False, compress=False):
         """Import .vcf files as VariantDataset
```
Comment (cseed) [inline python/hail/context.py]: VCF files.

File: python/hail/context.py [hunk 2]
```diff
@@ -355,6 +355,20 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
...
 
+        Since Hail's genotype representation does not yet support ploidy other than 2,
+        this module imports haploid genotypes as diploid. Hail fills in missing indices
```
Comment (cseed) [inline python/hail/context.py]: This is not a module.  That means something special in python.

File: python/hail/context.py [hunk 3]
```diff
@@ -355,6 +355,20 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
...
+
+        .. code-block:: text
+
+            Haploid:     1:0,6:7:70:70,0
+            Imported as: 1/1:0,6:7:70:70,1000,0
+
+            Haploid:     2:0,0,9:9:24:24,40,0
+            Imported as: 2/2:0,0,9:9:24:24,1000,40,1000:1000:0
+
         :param path: .vcf files to read.
```
Comment (cseed) [inline python/hail/context.py]: VCF files.

**Comment by tpoterba** [comment]
Back to you.

**Comment by lescai** [comment]
Hi guys,
I've cloned the master and rebuilt, but unfortunately I get the same error when annotating the VCF just read.
```
[Stage 0:====================================================>(2111 + 1) / 2112]hail: info: Coerced sorted dataset

------------Annotate the VCF file-------------
[Stage 1:>                                                     (0 + 160) / 2112]
[Stage 1:>                                                     (0 + 160) / 2112]found fatal is.hail.utils.package$FatalExcepti
on: swe.vcf.bgz: invalid AD field `24,0,0': expected 2 values, but got 3.
  offending line: 1	65684548	.	G	A	3524.14	VQSRTrancheSNP99.60to99.80	AC=1;AF=...
```

**Comment by tpoterba** [comment]
This doesn't look like the same error.  This error is found in a genotype call in a biallelic variant with 3 AD values -- a violation of the VCF spec (AD is "R"-numbered).  

We've seen this before on VCFs that were split by certain tools, and since there were enough of them, we added an option `skip_bad_ad` on import_vcf: https://hail.is/hail/hail.HailContext.html#hail.HailContext.import_vcf

You'll want to run with that option set to true.

**Pull request merged**